src/recipe.js

```diff
@@ -1,16 +1,32 @@
 class Recipe {
   constructor(recipe) {
-    this.id = recipe.id;
     this.name = recipe.name;
+    this.id = recipe.id;
     this.image = recipe.image;
-    this.tags = recipe.tags;
```

Comment:
I would like to know how the tags are going to be adjusted to search by the tags in the future